### PR TITLE
Switching content back to markdown

### PR DESCRIPTION
**Pull request by peteryates**
This is linked to dfe-digital/get-into-teaching-app#459

~~Here we're just working with a 'backup' story that isn't actually linked to by the site for experimentation purposes.~~

### CODE REVIEW

**Comment by peteryates** [comment]
This needs to stay in draft until https://github.com/DFE-Digital/get-into-teaching-app/pull/459 has been merged. The stories section will go mighty wrong if this goes in first :sweat_smile:

**Review by dbradford78 — APPROVED** [review]
Looks good

File: content/life-as-a-teacher/my-story-into-teaching/career-progression/grasp-every-opportunity.md [hunk 1]
```diff
@@ -1,84 +1,46 @@
...
+###Take on a challenge
+When I started teaching as a newly qualified teacher, I looked for opportunities to support the school and to enhance my experience.
+
+For example, in my first year, I led on re-designing the KS3 curriculum, and in my second year I started to become involved in the school's virtual learning environment. All of this added to my experience and provided me with plenty to talk about when applying for new roles.
+
+My long-term plan is to go into headship, but do not want to rush this. I want to be able to experience a range of different areas in schools to ensure that I am fully prepared for the challenges that a headship role will bring.
+
+I feel fortunate to be on the leadership salary, but I don't do the job for the money – for me it's about supporting students as a teacher and leader and making sure they achieve their full potential.
+
+
```
Comment (jebw) [inline content/life-as-a-teacher/my-story-into-teaching/career-progression/grasp-every-opportunity.md:40]: Note the end of the world but excess whitespace

File: content/life-as-a-teacher/my-story-into-teaching/career-progression/grasp-every-opportunity.md [hunk 2]
```diff
@@ -1,84 +1,46 @@
...
-         </div>
-    </div>
-</div>
+I always wanted to work in an environment that was never going to be the same every day – teaching definitely provides this. Working with students from different backgrounds and preparing them to take their next steps towards success was always a huge draw to the profession.
+
+As a teacher, you'll have the opportunity to develop and succeed in your career to realise your professional and personal ambitions. Before I was assistant headteacher I enjoyed a number of different roles learning new skills that have helped me to move forward in my career.
+
+> There are plenty of progression opportunities in teaching – you just have to grasp them.
+
+###Take on a challenge
```
Comment (jebw) [inline content/life-as-a-teacher/my-story-into-teaching/career-progression/grasp-every-opportunity.md:31]: Can't remember if Kramdown allows it but its a bit easier to read with a space in it `### Take on a challenger`

**Review by jebw — APPROVED** [review]
Wow - dramatic improvement.

Looks really good

File: content/life-as-a-teacher/my-story-into-teaching/career-progression/grasp-every-opportunity.md (hunk 2 above)
Comment (peteryates) [reply content/life-as-a-teacher/my-story-into-teaching/career-progression/grasp-every-opportunity.md:31]: Agree, will revisit and tidy.

**Pull request merged**